openapi/p4pa-pu-bff.openapi.yaml

```
@@ -269,6 +269,76 @@ paths:
         '500':
           description: Internal Server Error
       x-spring-paginated: true
+  /debt-position-types/receipts:
+    get:
+      tags:
+        - Receipts
+      summary: "Retrieve a page of receipts based on filters"
+      operationId: getReceipts
+      parameters:
+        - name: organizationId
+          in: query
+          required: true
+          schema:
+            type: integer
+            format: int64
+        - name: receiptOrigin
+          in: query
+          required: true
+          schema:
+            type: 'ReceiptOriginType'
+        - name: operatorExternalUserId
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: iuv
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: iur
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: iud
+          in: query
+          required: false
+          schema:
+            type: string
+        - name: debtPositionTypeOrgId
+          in: query
+          required: false
+          schema:
+            type: integer
+            format: int64
+        - name: fromDate
+          in: query
+          required: false
+          schema:
+            type: string
+            format: date-time
+        - name: toDate
+          in: query
+          required: false
+          schema:
+            type: string
+            format: date-time
```

```suggestion
        - name: paymentDateTimeFrom
          in: query
          required: false
          schema:
            type: string
            format: date-time
        - name: paymentDateTimeTo
          in: query
          required: false
          schema:
            type: string
            format: date-time
```

I'm renaming debt-position API parameters name according to this suggested change